cleverpush/src/main/java/com/cleverpush/stories/models/Story.java

```diff
@@ -0,0 +1,64 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class Story implements Serializable {
+
+    @SerializedName("_id")
+    @Expose
+    private String id;
+    @SerializedName("channel")
+    @Expose
+    private String channel;
+    @SerializedName("title")
+    @Expose
+    private String title;
+    @SerializedName("content")
+    @Expose
+    private Content content;
+    private boolean isOpened = false;
+
+    public String getId() {
+        return id;
```

Comment:
please use this in return, as you want to reference to the class scope variable (make it more descriptive)